app/src/main/java/appricottsoftware/clarity/LoginActivity.java

```diff
@@ -133,7 +134,38 @@ public void onClick(View v) {
 
     // HTTPS GET function to authenticate user. Currently not working.
     public boolean isAuthenticated(String email, String password) {
-        return true;
+        ClarityApp.getRestClient().authenticateUser(email, password, this, new JsonHttpResponseHandler() {
```

Comment:
We should probably get together and think about what needs to happen for each of these methods.

Reply:
Agreed!